[2025-07-16T15:37:40.573+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-16T15:37:40.610+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [queued]>
[2025-07-16T15:37:40.620+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [queued]>
[2025-07-16T15:37:40.621+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-07-16T15:37:40.634+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): log_pipeline_start> on 2025-07-15 00:00:00+00:00
[2025-07-16T15:37:40.642+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=203) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-16T15:37:40.643+0000] {standard_task_runner.py:63} INFO - Started process 204 to run task
[2025-07-16T15:37:40.643+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'bank_data_pipeline', 'log_pipeline_start', 'scheduled__2025-07-15T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/bank_data_pipeline.py', '--cfg-path', '/tmp/tmpqpcafp9m']
[2025-07-16T15:37:40.645+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask log_pipeline_start
[2025-07-16T15:37:40.661+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-16T15:37:40.700+0000] {task_command.py:426} INFO - Running <TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [running]> on host 0cebe846fd9f
[2025-07-16T15:37:40.792+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data_engineering_team' AIRFLOW_CTX_DAG_ID='bank_data_pipeline' AIRFLOW_CTX_TASK_ID='log_pipeline_start' AIRFLOW_CTX_EXECUTION_DATE='2025-07-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-15T00:00:00+00:00'
[2025-07-16T15:37:40.793+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-16T15:37:40.805+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-16T15:37:40.806+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/bank_data_pipeline.py", line 66, in log_task_start
    logger = PipelineLogger()
             ^^^^^^^^^^^^^^
NameError: name 'PipelineLogger' is not defined
[2025-07-16T15:37:40.821+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=bank_data_pipeline, task_id=log_pipeline_start, run_id=scheduled__2025-07-15T00:00:00+00:00, execution_date=20250715T000000, start_date=20250716T153740, end_date=20250716T153740
[2025-07-16T15:37:40.830+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 3 for task log_pipeline_start (name 'PipelineLogger' is not defined; 204)
[2025-07-16T15:37:40.857+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-07-16T15:37:40.896+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-16T15:37:40.898+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-07-16T16:11:15.117+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-16T16:11:15.149+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [queued]>
[2025-07-16T16:11:15.156+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [queued]>
[2025-07-16T16:11:15.156+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-07-16T16:11:15.167+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): log_pipeline_start> on 2025-07-15 00:00:00+00:00
[2025-07-16T16:11:15.174+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=236) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-16T16:11:15.175+0000] {standard_task_runner.py:63} INFO - Started process 237 to run task
[2025-07-16T16:11:15.174+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'bank_data_pipeline', 'log_pipeline_start', 'scheduled__2025-07-15T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/bank_data_pipeline.py', '--cfg-path', '/tmp/tmp9ydnz149']
[2025-07-16T16:11:15.176+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask log_pipeline_start
[2025-07-16T16:11:15.189+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-16T16:11:15.235+0000] {task_command.py:426} INFO - Running <TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [running]> on host 3f78294f0070
[2025-07-16T16:11:15.325+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data_engineering_team' AIRFLOW_CTX_DAG_ID='bank_data_pipeline' AIRFLOW_CTX_TASK_ID='log_pipeline_start' AIRFLOW_CTX_EXECUTION_DATE='2025-07-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-15T00:00:00+00:00'
[2025-07-16T16:11:15.325+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-16T16:11:15.344+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2025-07-16T16:11:15.344+0000] {logging_mixin.py:188} INFO - 2025-07-16 16:11:15,344 - pipeline_logger - ERROR - Failed to log task start: can't adapt type 'Proxy'
[2025-07-16T16:11:15.344+0000] {pipeline_logger.py:79} ERROR - Failed to log task start: can't adapt type 'Proxy'
[2025-07-16T16:11:15.345+0000] {logging_mixin.py:188} INFO - âœ… Task log_pipeline_start started for run scheduled__2025-07-15T00:00:00+00:00
[2025-07-16T16:11:15.345+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-16T16:11:15.345+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-16T16:11:15.355+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=bank_data_pipeline, task_id=log_pipeline_start, run_id=scheduled__2025-07-15T00:00:00+00:00, execution_date=20250715T000000, start_date=20250716T161115, end_date=20250716T161115
[2025-07-16T16:11:15.388+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-16T16:11:15.431+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-16T16:11:15.433+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-07-16T16:20:15.818+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-16T16:20:15.853+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [queued]>
[2025-07-16T16:20:15.861+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [queued]>
[2025-07-16T16:20:15.862+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-07-16T16:20:15.872+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): log_pipeline_start> on 2025-07-15 00:00:00+00:00
[2025-07-16T16:20:15.880+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=203) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-16T16:20:15.881+0000] {standard_task_runner.py:63} INFO - Started process 204 to run task
[2025-07-16T16:20:15.880+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'bank_data_pipeline', 'log_pipeline_start', 'scheduled__2025-07-15T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/bank_data_pipeline.py', '--cfg-path', '/tmp/tmpslpindo_']
[2025-07-16T16:20:15.882+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask log_pipeline_start
[2025-07-16T16:20:15.898+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-16T16:20:15.937+0000] {task_command.py:426} INFO - Running <TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [running]> on host 3a97a985f8ee
[2025-07-16T16:20:16.036+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data_engineering_team' AIRFLOW_CTX_DAG_ID='bank_data_pipeline' AIRFLOW_CTX_TASK_ID='log_pipeline_start' AIRFLOW_CTX_EXECUTION_DATE='2025-07-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-15T00:00:00+00:00'
[2025-07-16T16:20:16.037+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-16T16:20:16.063+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2025-07-16T16:20:16.064+0000] {logging_mixin.py:188} INFO - 2025-07-16 16:20:16,064 - pipeline_logger - ERROR - Failed to log task start: can't adapt type 'Proxy'
[2025-07-16T16:20:16.064+0000] {pipeline_logger.py:79} ERROR - Failed to log task start: can't adapt type 'Proxy'
[2025-07-16T16:20:16.064+0000] {logging_mixin.py:188} INFO - âœ… Task log_pipeline_start started for run scheduled__2025-07-15T00:00:00+00:00
[2025-07-16T16:20:16.064+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-16T16:20:16.064+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-16T16:20:16.078+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=bank_data_pipeline, task_id=log_pipeline_start, run_id=scheduled__2025-07-15T00:00:00+00:00, execution_date=20250715T000000, start_date=20250716T162015, end_date=20250716T162016
[2025-07-16T16:20:16.094+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-16T16:20:16.137+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-16T16:20:16.139+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-07-16T16:23:08.058+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-16T16:23:08.098+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [queued]>
[2025-07-16T16:23:08.107+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [queued]>
[2025-07-16T16:23:08.108+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-07-16T16:23:08.119+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): log_pipeline_start> on 2025-07-15 00:00:00+00:00
[2025-07-16T16:23:08.127+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=226) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-16T16:23:08.128+0000] {standard_task_runner.py:63} INFO - Started process 227 to run task
[2025-07-16T16:23:08.129+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'bank_data_pipeline', 'log_pipeline_start', 'scheduled__2025-07-15T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/bank_data_pipeline.py', '--cfg-path', '/tmp/tmpsqsv8tj7']
[2025-07-16T16:23:08.132+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask log_pipeline_start
[2025-07-16T16:23:08.155+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-16T16:23:08.190+0000] {task_command.py:426} INFO - Running <TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [running]> on host fee2a1e1d73e
[2025-07-16T16:23:08.268+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data_engineering_team' AIRFLOW_CTX_DAG_ID='bank_data_pipeline' AIRFLOW_CTX_TASK_ID='log_pipeline_start' AIRFLOW_CTX_EXECUTION_DATE='2025-07-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-15T00:00:00+00:00'
[2025-07-16T16:23:08.269+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-16T16:23:08.288+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2025-07-16T16:23:08.289+0000] {logging_mixin.py:188} INFO - 2025-07-16 16:23:08,289 - pipeline_logger - ERROR - Failed to log task start: can't adapt type 'Proxy'
[2025-07-16T16:23:08.289+0000] {pipeline_logger.py:79} ERROR - Failed to log task start: can't adapt type 'Proxy'
[2025-07-16T16:23:08.289+0000] {logging_mixin.py:188} INFO - âœ… Task log_pipeline_start started for run scheduled__2025-07-15T00:00:00+00:00
[2025-07-16T16:23:08.289+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-16T16:23:08.290+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-16T16:23:08.303+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=bank_data_pipeline, task_id=log_pipeline_start, run_id=scheduled__2025-07-15T00:00:00+00:00, execution_date=20250715T000000, start_date=20250716T162308, end_date=20250716T162308
[2025-07-16T16:23:08.342+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-16T16:23:08.376+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-16T16:23:08.378+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-07-16T16:29:42.645+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-16T16:29:42.679+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [queued]>
[2025-07-16T16:29:42.687+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [queued]>
[2025-07-16T16:29:42.687+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-07-16T16:29:42.698+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): log_pipeline_start> on 2025-07-15 00:00:00+00:00
[2025-07-16T16:29:42.706+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=203) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-16T16:29:42.707+0000] {standard_task_runner.py:63} INFO - Started process 204 to run task
[2025-07-16T16:29:42.707+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'bank_data_pipeline', 'log_pipeline_start', 'scheduled__2025-07-15T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/bank_data_pipeline.py', '--cfg-path', '/tmp/tmpa8xtpawh']
[2025-07-16T16:29:42.710+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask log_pipeline_start
[2025-07-16T16:29:42.732+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-16T16:29:42.775+0000] {task_command.py:426} INFO - Running <TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [running]> on host 07a1f2c7fa0b
[2025-07-16T16:29:42.868+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data_engineering_team' AIRFLOW_CTX_DAG_ID='bank_data_pipeline' AIRFLOW_CTX_TASK_ID='log_pipeline_start' AIRFLOW_CTX_EXECUTION_DATE='2025-07-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-15T00:00:00+00:00'
[2025-07-16T16:29:42.869+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-16T16:29:42.887+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2025-07-16T16:29:42.887+0000] {logging_mixin.py:188} INFO - 2025-07-16 16:29:42,887 - pipeline_logger - ERROR - Failed to log task start: can't adapt type 'Proxy'
[2025-07-16T16:29:42.887+0000] {pipeline_logger.py:79} ERROR - Failed to log task start: can't adapt type 'Proxy'
[2025-07-16T16:29:42.888+0000] {logging_mixin.py:188} INFO - âœ… Task log_pipeline_start started for run scheduled__2025-07-15T00:00:00+00:00
[2025-07-16T16:29:42.888+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-16T16:29:42.889+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-16T16:29:42.900+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=bank_data_pipeline, task_id=log_pipeline_start, run_id=scheduled__2025-07-15T00:00:00+00:00, execution_date=20250715T000000, start_date=20250716T162942, end_date=20250716T162942
[2025-07-16T16:29:42.920+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-16T16:29:42.960+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-16T16:29:42.962+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-07-16T16:43:36.718+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-16T16:43:36.753+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [queued]>
[2025-07-16T16:43:36.760+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [queued]>
[2025-07-16T16:43:36.761+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-07-16T16:43:36.771+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): log_pipeline_start> on 2025-07-15 00:00:00+00:00
[2025-07-16T16:43:36.779+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=391) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-16T16:43:36.780+0000] {standard_task_runner.py:63} INFO - Started process 392 to run task
[2025-07-16T16:43:36.781+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'bank_data_pipeline', 'log_pipeline_start', 'scheduled__2025-07-15T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/bank_data_pipeline.py', '--cfg-path', '/tmp/tmp6pw568pl']
[2025-07-16T16:43:36.783+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask log_pipeline_start
[2025-07-16T16:43:36.800+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-16T16:43:36.847+0000] {task_command.py:426} INFO - Running <TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [running]> on host 6033db535fcd
[2025-07-16T16:43:36.933+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data_engineering_team' AIRFLOW_CTX_DAG_ID='bank_data_pipeline' AIRFLOW_CTX_TASK_ID='log_pipeline_start' AIRFLOW_CTX_EXECUTION_DATE='2025-07-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-15T00:00:00+00:00'
[2025-07-16T16:43:36.935+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-16T16:43:36.954+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2025-07-16T16:43:36.954+0000] {logging_mixin.py:188} INFO - 2025-07-16 16:43:36,954 - pipeline_logger - ERROR - Failed to log task start: can't adapt type 'Proxy'
[2025-07-16T16:43:36.954+0000] {pipeline_logger.py:79} ERROR - Failed to log task start: can't adapt type 'Proxy'
[2025-07-16T16:43:36.955+0000] {logging_mixin.py:188} INFO - âœ… Task log_pipeline_start started for run scheduled__2025-07-15T00:00:00+00:00
[2025-07-16T16:43:36.955+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-16T16:43:36.955+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-16T16:43:36.965+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=bank_data_pipeline, task_id=log_pipeline_start, run_id=scheduled__2025-07-15T00:00:00+00:00, execution_date=20250715T000000, start_date=20250716T164336, end_date=20250716T164336
[2025-07-16T16:43:36.993+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-16T16:43:37.050+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-16T16:43:37.052+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-07-16T16:46:29.601+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-16T16:46:29.633+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [queued]>
[2025-07-16T16:46:29.641+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [queued]>
[2025-07-16T16:46:29.641+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-07-16T16:46:29.651+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): log_pipeline_start> on 2025-07-15 00:00:00+00:00
[2025-07-16T16:46:29.659+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=219) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-16T16:46:29.660+0000] {standard_task_runner.py:63} INFO - Started process 220 to run task
[2025-07-16T16:46:29.663+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'bank_data_pipeline', 'log_pipeline_start', 'scheduled__2025-07-15T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/bank_data_pipeline.py', '--cfg-path', '/tmp/tmp26vklfrj']
[2025-07-16T16:46:29.664+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask log_pipeline_start
[2025-07-16T16:46:29.678+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-16T16:46:29.717+0000] {task_command.py:426} INFO - Running <TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [running]> on host f8c2cd69d5df
[2025-07-16T16:46:29.804+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data_engineering_team' AIRFLOW_CTX_DAG_ID='bank_data_pipeline' AIRFLOW_CTX_TASK_ID='log_pipeline_start' AIRFLOW_CTX_EXECUTION_DATE='2025-07-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-15T00:00:00+00:00'
[2025-07-16T16:46:29.805+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-16T16:46:29.826+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2025-07-16T16:46:29.827+0000] {logging_mixin.py:188} INFO - 2025-07-16 16:46:29,827 - pipeline_logger - ERROR - Failed to log task start: can't adapt type 'Proxy'
[2025-07-16T16:46:29.827+0000] {pipeline_logger.py:79} ERROR - Failed to log task start: can't adapt type 'Proxy'
[2025-07-16T16:46:29.827+0000] {logging_mixin.py:188} INFO - âœ… Task log_pipeline_start started for run scheduled__2025-07-15T00:00:00+00:00
[2025-07-16T16:46:29.827+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-16T16:46:29.827+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-16T16:46:29.841+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=bank_data_pipeline, task_id=log_pipeline_start, run_id=scheduled__2025-07-15T00:00:00+00:00, execution_date=20250715T000000, start_date=20250716T164629, end_date=20250716T164629
[2025-07-16T16:46:29.874+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-16T16:46:29.923+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-16T16:46:29.925+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-07-16T16:51:36.063+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-16T16:51:36.094+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [queued]>
[2025-07-16T16:51:36.104+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [queued]>
[2025-07-16T16:51:36.104+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-07-16T16:51:36.117+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): log_pipeline_start> on 2025-07-15 00:00:00+00:00
[2025-07-16T16:51:36.122+0000] {standard_task_runner.py:63} INFO - Started process 267 to run task
[2025-07-16T16:51:36.127+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'bank_data_pipeline', 'log_pipeline_start', 'scheduled__2025-07-15T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/bank_data_pipeline.py', '--cfg-path', '/tmp/tmpwndm18n1']
[2025-07-16T16:51:36.130+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask log_pipeline_start
[2025-07-16T16:51:36.149+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-16T16:51:36.186+0000] {task_command.py:426} INFO - Running <TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [running]> on host 039da2dd7cbf
[2025-07-16T16:51:36.269+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data_engineering_team' AIRFLOW_CTX_DAG_ID='bank_data_pipeline' AIRFLOW_CTX_TASK_ID='log_pipeline_start' AIRFLOW_CTX_EXECUTION_DATE='2025-07-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-15T00:00:00+00:00'
[2025-07-16T16:51:36.269+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-16T16:51:36.287+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/utils/context.py:320 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2025-07-16T16:51:36.287+0000] {logging_mixin.py:188} INFO - 2025-07-16 16:51:36,287 - pipeline_logger - ERROR - Failed to log task start: can't adapt type 'Proxy'
[2025-07-16T16:51:36.287+0000] {pipeline_logger.py:79} ERROR - Failed to log task start: can't adapt type 'Proxy'
[2025-07-16T16:51:36.287+0000] {logging_mixin.py:188} INFO - âœ… Task log_pipeline_start started for run scheduled__2025-07-15T00:00:00+00:00
[2025-07-16T16:51:36.288+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-16T16:51:36.288+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-16T16:51:36.299+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=bank_data_pipeline, task_id=log_pipeline_start, run_id=scheduled__2025-07-15T00:00:00+00:00, execution_date=20250715T000000, start_date=20250716T165136, end_date=20250716T165136
[2025-07-16T16:51:36.337+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-16T16:51:36.371+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-16T16:51:36.372+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-07-16T16:57:37.237+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-16T16:57:37.269+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [queued]>
[2025-07-16T16:57:37.276+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [queued]>
[2025-07-16T16:57:37.277+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-07-16T16:57:37.288+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): log_pipeline_start> on 2025-07-15 00:00:00+00:00
[2025-07-16T16:57:37.294+0000] {standard_task_runner.py:63} INFO - Started process 220 to run task
[2025-07-16T16:57:37.297+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'bank_data_pipeline', 'log_pipeline_start', 'scheduled__2025-07-15T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/bank_data_pipeline.py', '--cfg-path', '/tmp/tmppv0az_1n']
[2025-07-16T16:57:37.299+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask log_pipeline_start
[2025-07-16T16:57:37.316+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-16T16:57:37.356+0000] {task_command.py:426} INFO - Running <TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [running]> on host 130d0664bbcb
[2025-07-16T16:57:37.445+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data_engineering_team' AIRFLOW_CTX_DAG_ID='bank_data_pipeline' AIRFLOW_CTX_TASK_ID='log_pipeline_start' AIRFLOW_CTX_EXECUTION_DATE='2025-07-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-15T00:00:00+00:00'
[2025-07-16T16:57:37.446+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-16T16:57:37.463+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/utils/context.py:320 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2025-07-16T16:57:37.463+0000] {logging_mixin.py:188} INFO - 2025-07-16 16:57:37,463 - pipeline_logger - ERROR - Failed to log task start: can't adapt type 'Proxy'
[2025-07-16T16:57:37.463+0000] {pipeline_logger.py:79} ERROR - Failed to log task start: can't adapt type 'Proxy'
[2025-07-16T16:57:37.464+0000] {logging_mixin.py:188} INFO - âœ… Task log_pipeline_start started for run scheduled__2025-07-15T00:00:00+00:00
[2025-07-16T16:57:37.464+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-16T16:57:37.465+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-16T16:57:37.474+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=bank_data_pipeline, task_id=log_pipeline_start, run_id=scheduled__2025-07-15T00:00:00+00:00, execution_date=20250715T000000, start_date=20250716T165737, end_date=20250716T165737
[2025-07-16T16:57:37.509+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-16T16:57:37.550+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-16T16:57:37.551+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-07-16T17:04:16.984+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-16T17:04:17.010+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [queued]>
[2025-07-16T17:04:17.018+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [queued]>
[2025-07-16T17:04:17.018+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-07-16T17:04:17.028+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): log_pipeline_start> on 2025-07-15 00:00:00+00:00
[2025-07-16T17:04:17.034+0000] {standard_task_runner.py:63} INFO - Started process 249 to run task
[2025-07-16T17:04:17.039+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'bank_data_pipeline', 'log_pipeline_start', 'scheduled__2025-07-15T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/bank_data_pipeline.py', '--cfg-path', '/tmp/tmpj44x30wt']
[2025-07-16T17:04:17.042+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask log_pipeline_start
[2025-07-16T17:04:17.059+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-16T17:04:17.101+0000] {task_command.py:426} INFO - Running <TaskInstance: bank_data_pipeline.log_pipeline_start scheduled__2025-07-15T00:00:00+00:00 [running]> on host 055183203c9b
[2025-07-16T17:04:17.199+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data_engineering_team' AIRFLOW_CTX_DAG_ID='bank_data_pipeline' AIRFLOW_CTX_TASK_ID='log_pipeline_start' AIRFLOW_CTX_EXECUTION_DATE='2025-07-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-15T00:00:00+00:00'
[2025-07-16T17:04:17.199+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-16T17:04:17.219+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/utils/context.py:320 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2025-07-16T17:04:17.219+0000] {logging_mixin.py:188} INFO - 2025-07-16 17:04:17,219 - pipeline_logger - ERROR - Failed to log task start: can't adapt type 'Proxy'
[2025-07-16T17:04:17.219+0000] {pipeline_logger.py:79} ERROR - Failed to log task start: can't adapt type 'Proxy'
[2025-07-16T17:04:17.220+0000] {logging_mixin.py:188} INFO - âœ… Task log_pipeline_start started for run scheduled__2025-07-15T00:00:00+00:00
[2025-07-16T17:04:17.220+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-16T17:04:17.220+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-16T17:04:17.231+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=bank_data_pipeline, task_id=log_pipeline_start, run_id=scheduled__2025-07-15T00:00:00+00:00, execution_date=20250715T000000, start_date=20250716T170417, end_date=20250716T170417
[2025-07-16T17:04:17.248+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-16T17:04:17.293+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-16T17:04:17.295+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
